src/main/java/uk/gov/cabinetoffice/csl/service/IdentityService.java

```diff
@@ -38,6 +38,7 @@ public class IdentityService {
     private final IdentityRepository identityRepository;
     private final CompoundRoles compoundRoles;
     private final ICivilServantRegistryClient civilServantRegistryClient;
```

Comment:
civilServantRegistryClient reference may not be required any more because CsrsService is used now.